<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputs_of_task</class>
 <widget class="QWidget" name="inputs_of_task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="settimeanddate_groupbox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>571</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Set Date and Time of this task</string>
   </property>
   <widget class="QLabel" name="settime_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>set the time of your topic in hh:mm:ss mode:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="settime_lineedit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="setdate_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>select the day of this task :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_calender_pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>show the calender</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>511</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>WARNING! At the first click on save and then click on show the calender!</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="save_pushbutton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="setdescription_groupbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>set the Description of this task</string>
   </property>
   <widget class="QLabel" name="description_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="description_textedit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>104</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="setpriority_groupbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>set the Priority of this task</string>
   </property>
   <widget class="QComboBox" name="priority_combobox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="selectpriority_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>select the Priority of this task</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="settopic_groupbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>set the Topic of this task</string>
   </property>
   <widget class="QLabel" name="topic_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Topic:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="topic_lineedit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Quit_pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Quit_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>inputs_of_task</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
